VideoPlayer/Android/testplayerplugin-android/src/main/java/com/applicaster/player/plugins/testplayerplugin_android/TestPlayerActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.applicaster.player.plugins.testplayerplugin_android;
+
+import android.media.MediaPlayer;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.MediaController;
+import android.widget.VideoView;
+
+import com.applicaster.plugin_manager.playersmanager.Playable;
+
+import static com.applicaster.player.plugins.testplayerplugin_android.TestPlayerAdapter.KEY_PLAYABLE;
+
+public class TestPlayerActivity extends AppCompatActivity {
+
+    private Playable playable;
+    private VideoView videoView;
+    private MediaController mediaController;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        requestWindowFeature(Window.FEATURE_NO_TITLE);
+        getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
+                WindowManager.LayoutParams.FLAG_FULLSCREEN);
+
+        if (getIntent().getExtras() != null) {
+            playable = (Playable) getIntent().getExtras()
+                    .getSerializable(KEY_PLAYABLE);
+        }
+
+        setContentView(R.layout.activity_test_player);
+        videoView = findViewById(R.id.video_view);
+        hideSystemUI();
+    }
+
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+        if (playable != null) {
+            videoView.setVideoURI(Uri.parse(playable.getContentVideoURL()));
+            videoView.start();
+            videoView.setOnPreparedListener(new MediaPlayer.OnPreparedListener() {
+                @Override
+                public void onPrepared(MediaPlayer mediaPlayer) {
+                    mediaPlayer.start();
+                    mediaController = new MediaController(TestPlayerActivity.this);
```

Comment:
Can this get called multiple times during a video playback?

Reply:
yes it can